wiki/Medals/en.md

```diff
@@ -158,140 +38,20 @@ All Mod Introduction medals are eligible to be unlocked in any [game mode](/wiki
 
 As the Hush-Hush medals are meant to be secret achievements of some sort, the list below only note out the *descriptions and flair texts* of the medals instead of the exact method of getting them; however, players are [welcome to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```

Comment:
```suggestion
As the Hush-Hush medals are meant to be secret achievements of some sort, the list below only notes out the *descriptions and flair texts* of the medals instead of the exact method of getting them; however, players are [welcome to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```